src/components/Common/FolderContextMenu.tsx

```diff
@@ -65,7 +111,7 @@ export const FolderContextMenu = ({
         onClick: onAddFolder,
       },
     ],
-    [t, onRename, onOpenShareModal, onDelete, onAddFolder],
+    [t, onRename, onShare, onDelete, onAddFolder],
```

Comment:
It said - React Hook useMemo has missing dependencies: 'folder.isPublished', 'isPublishingEnabled', 'isSharingEnabled', 'onPublish', 'onPublishUpdate', and 'onUnpublish'. Either include them or remove the dependency array
Do you want to fix it?